yi-core/yi-core.cabal

```diff
@@ -135,3 +136,54 @@ library
       System.CanonicalizePath
       Yi.Buffer.Implementation
       Paths_yi_core
+
+test-suite tasty
```

Comment:
The global dependencies (Line 11-47 in the package.yaml) are also added to the dependencies of the tests I think. Maybe I have a newer version of hpack installed that does that.